app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/radioGroupControl.jsx

```diff
@@ -24,9 +24,11 @@ import { FIELD_LABEL_WIDTH } from './constants';
 
 const cx = classNames.bind(style);
 
-export const RadioGroupControl = ({ onChange, value, options, fieldLabel }) => {
+export const RadioGroupControl = ({ onChange, value, options, fieldLabel, onOptionChange }) => {
   const handleValueChange = ({ target: { value: radioButtonValue } }) => {
-    onChange(JSON.parse(radioButtonValue));
+    const parsedValue = JSON.parse(radioButtonValue);
+    onOptionChange(parsedValue);
+    onChange(parsedValue);
```

Comment:
In my opinion it's confusing why we have `onOptionChange` and `onChange` they both represent the same action and at the first glance you cannot define where it analytics and where it real change fn I guess better combine them